salt/sdb/etcd_db.py

```diff
@@ -7,6 +7,8 @@
 :depends:       python-etcd
 :platform:      all
 
+.. versionadded:: 2015.2
+
 This module allows access to the etcd database using an ``sdb://`` URI. This
```

Comment:
Oops, sorry about that. It was written when 2014.1 was current, but I failed to commit it until today.